src/main/java/frc/robot/subsystems/climb/ClimbSubsystem.java

```diff
@@ -82,6 +101,27 @@ public ClimbSubsystem(ClimbIO io) {
 
     climbMachine =
         new StateMachine<ClimbState, ClimbAction>(climbMachineConfiguration, ClimbState.IDLE);
+
+    climbkP =
+        new LoggedTunableNumber("climbTunables/climbkP", ClimbConstants.synced.getObject().climbkP);
+    climbkI =
+        new LoggedTunableNumber("climbTunables/climbkI", ClimbConstants.synced.getObject().climbkI);
+    climbkD =
+        new LoggedTunableNumber("climbTunables/climbkD", ClimbConstants.synced.getObject().climbkD);
+
+    climbkS =
+        new LoggedTunableNumber("climbTunables/climbkS", ClimbConstants.synced.getObject().climbkS);
+    climbkV =
+        new LoggedTunableNumber("climbTunables/climbkV", ClimbConstants.synced.getObject().climbkV);
+    climbkA =
+        new LoggedTunableNumber("climbTunables/climbkA", ClimbConstants.synced.getObject().climbkA);
+    climbkG =
+        new LoggedTunableNumber("climbTunables/climbkG", ClimbConstants.synced.getObject().climbkG);
```

Comment:
See my other comment about updating the JSON files to use the correct names for the pid gains